### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ia-lerimund**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentPrice = 40;
+  const threeDaysDiscount = 20;
+  const SevenDaysDiscount = 50;
```
Comment (OlEzhA0) [inline src/calculateRentalCost.js:34]: don't create constant in `camel case` from `uppercase` letter

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hey! Check please my comment

File: src/calculateRentalCost.js (hunk 1 above)
Comment (ia-lerimund) [reply src/calculateRentalCost.js]: sorry, made a typo. now fixed. thanks!

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentPrice = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  if (days <= 2) {
+    return rentPrice * days;
```
Comment (Anastasiia-Svintsova) [inline src/calculateRentalCost.js:37]: You use `rentPrice * days` 3 times, so move it to variable

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Good! One small fix needed

File: src/calculateRentalCost.js (hunk 2 above)
Comment (ia-lerimund) [reply src/calculateRentalCost.js]: I've changed this, thank you!

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**